src/seedu/addressbook/data/person/Email.java

```diff
@@ -38,6 +38,11 @@ public static boolean isValidEmail(String test) {
     }
 
     @Override
+    public String getPrintableString(){
+        return this.value;
```

Comment:
The method performs the same action as `toString`. Use a better implementation if you wish to implement the `Printable` interface.